x-pack/plugins/ingest_manager/server/routes/epm/handlers.ts

```diff
@@ -146,10 +148,12 @@ export const getInfoHandler: RequestHandler<TypeOf<typeof GetInfoRequestSchema.p
 export const installPackageHandler: RequestHandler<TypeOf<
   typeof InstallPackageRequestSchema.params
 >> = async (context, request, response) => {
+  const logger = appContextService.getLogger();
+  const savedObjectsClient = context.core.savedObjects.client;
+  const callCluster = context.core.elasticsearch.legacy.client.callAsCurrentUser;
+  const { pkgkey } = request.params;
+  const [pkgName, pkgVersion] = pkgkey.split('-');
```

Comment:
nit: this could probably be in a dedicated function

Reply:
thanks.  there is a PR to remove it when we change the api from /pkg-version/ to /pkg/version